README.md

```diff
@@ -107,3 +108,9 @@ DISABLE_DNAT={true | false}
 If there are any issues with processing files, the error logs of all micro-services can be downloaded and sent to Brighter AI for further investigation.
 
 To download the error logs, run `./download_logs.sh` and send the output zip file (`redact_logs.zip`) to Brighter AI
+
+#### Installing as a Service
+
+To install Redact Enterprise as a systemd service, use the `./install.sh` script and supply an installtion directory. For example, `sudo ./install.sh /etc` will install the systemd service and store the necessary files under `/etc`. To uninstall, run `./uninstall.sh`, also with the installtion directory as an argument.
```

Comment:
When trying to install I ran into the issue of the screenshot. Where did I do this? I created another VM (not the one we created together) and did the uninstall and install script.
![image](https://github.com/brighter-ai/redact-enterprise-orchestration/assets/28999441/d3a04f8f-28a5-4ac2-9dae-e10cf719bc99)
I think you mentioned this issue and is related to a GPU not being available. 

Nonetheless, we will need to provide further instructions on how to actually run the service. As defined [here](https://github.com/brighter-ai/redact-license-server/tree/main/scripts/systemd#starting-and-enabling-as-root) there is a major difference between `systemctl start redact.service` and `systemctl enable redact.service`. Once you enable a service, systemd takes actually care of starting the service when a restart is done. This is pretty much our major concern and goal of this PR.

To check if a systemd service is running and checked in for restarting one must do `systemctl status redact.service`

I would probably vote for creating a VM that has a GPU attached to test it properly. Testing it properly would then mean:
- install, start and enable service
- check service is running and enabled
- test redact service via a POST request
- restart the VM
- test redact service again via  POST request
- check service is running and enabled

I'm also happy to get involved and start to commit to this PR. Let me know if we should work together on this. I don't want to hijack your PR :)

Reply:
For your testing VM, you should disable secure boot. Hopefully this screenshot shows how to do so, but we can also have a call and do so together if needed. 
<img width="997" alt="Screenshot 2023-09-26 at 11 57 06" src="https://github.com/brighter-ai/redact-enterprise-orchestration/assets/70214205/58b5389d-c7dc-4c9a-b741-eb67726ab9e8">

To your second point: `systemctl enable redact.service` is also working, and I can update the README with these instructions as well. I've tested also how you described manually and all is working by my side. 

Would you also suggest we try and automate this testing process? I would see any automated testing infrastructure more as a part of one or both of these two tickets, so would be more in favour of extending the descriptions on these tickets and prioritising them with Nakul during replenishment / planning: 
https://brighterai.atlassian.net/browse/PHX-605?atlOrigin=eyJpIjoiZTc4OTYzMzZhNTkyNDM1NjhhMzdlZGYwMjJlZTc1MTgiLCJwIjoiaiJ9
https://brighterai.atlassian.net/browse/PHX-468?atlOrigin=eyJpIjoiYWE3M2M1OGY0ZjI5NGQ5OThhZmMxZjIwMDk5MGRlYjQiLCJwIjoiaiJ9
As part of this PR / ticket, maybe a confluence doc describing in detail how to manually test this makes most sense?